### PR TITLE
Feature/home dash

### PR DESCRIPTION
**Pull request by hez**

### CODE REVIEW

File: lib/brew_dash/database_brew_provider.ex [hunk 1]
```diff
@@ -0,0 +1,56 @@
+defmodule BrewDash.DatabaseBrewProvider do
+  use HomeDash.Provider
+  alias BrewDash.Brews
+
+  @displayable_sessions 8
+
+  def updated(server_name \\ __MODULE__), do: send(server_name, :updated_brews)
```
Comment (Baradoy) [inline lib/brew_dash/database_brew_provider.ex]: ❔ What is this for? I am not convinced it works.

File: lib/brew_dash/application.ex [hunk 2]
```diff
@@ -35,6 +36,10 @@ defmodule BrewDash.Application do
     defp migrate, do: nil
   end
 
+  defp home_dash_servers do
+    [{BrewDash.DatabaseBrewProvider, []}, {HomeDash.Providers.Welcome, []}]
```
Comment (Baradoy) [inline lib/brew_dash/application.ex]: ❔ Do we need the Welcome cards provider here at all?

**Review by Baradoy — APPROVED** [review]
Neat!

File: lib/brew_dash/application.ex (hunk 2 above)
Comment (hez) [reply lib/brew_dash/application.ex]: probably not, was using it for testing the home dash widgets as well

File: lib/brew_dash/database_brew_provider.ex (hunk 1 above)
Comment (hez) [reply lib/brew_dash/database_brew_provider.ex]: That looks like an old function from a previous incarnation, 🪓

**Pull request merged**